authorization-service/lambda/basicAuthorizer.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  APIGatewayAuthorizerResult,
+  APIGatewayRequestAuthorizerEvent,
+  Context,
+  Callback,
+} from "aws-lambda";
+
+export const handler = async (
```

Comment:
There are 2 ways of implementation of a handler:
1. async (which is used here)
2. passing `callback` function

There is no need to use both. I recommend to use `async` instead of `callback` to leverage modern ES spec.
In case of using `async` it's enough to return response object.